.travis.yml

```diff
@@ -5,14 +5,11 @@ sudo: false
 
 language: generic
 
-services:
-    - docker
-addons:
-  apt:
-    packages:
-      - docker-ce
-
-script: ./test-build.sh $NODE_VERSION $VARIANT
+before_install:
+  - source functions.sh; images_updated $TRAVIS_COMMIT_RANGE || tests_updated $TRAVIS_COMMIT_RANGE || travis_terminate
```

Comment:
Ah, so this tests for whether the images were updated _or_ if the test scripts have been updated? Nice.